.github/CONTRIBUTING.md

```diff
@@ -67,6 +67,37 @@ In addition to our automatically enforced [lint rules](https://github.com/bpmn-i
 
 __Rationale:__ People should be able to consume parts of the library with an ES module aware bundler such as [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org) without the need to use a transpiler such as [Babel](https://babeljs.io/).
 
+### Unit Tests
```

Comment:
We could link the test section here: https://github.com/bpmn-io/bpmn-js/pull/1374/files#diff-98e64bc1cd2db9333c6effe87bbe0d6dfe8714aba4c6bde45aa037fe0796e44cR112.

Reply:
The link you provided just links to my commit? I am not sure I get what you mean.

Reply:
Points to last section: 

```
...
* The code is tested
...
```

Reply:
:+1: Done